src/schema/schema.graphql

```diff
@@ -2002,6 +2019,12 @@ type Query {
 		folders: String,
 		includeGal: Boolean
 	): AutoCompleteResponse
+	autoCompleteGAL(
+		limit: Int
+		name: String
+		type: String
```

Comment:
this should be an enum, with below possible values
`all|account|resource|group`

Reply:
I have changed this to an enum but I do not know why it isn't reflected.